node/src/components/rpc_server/rpcs/state.rs

```diff
@@ -113,6 +113,7 @@ static QUERY_GLOBAL_STATE_RESULT: Lazy<QueryGlobalStateResult> =
         merkle_proof: MERKLE_PROOF.clone(),
     });
 static GET_TRIE_PARAMS: Lazy<GetTrieParams> = Lazy::new(|| GetTrieParams {
+    trie_index: 0, // TODO: get the index from somewhere?
```

Comment:
Should the RPC just always return the full data instead of chunks?

Reply:
If that's not too much, then why not. But can the RPC server handle such a message size?

Reply:
I don't know, but at least it wouldn't change the API for now.
We could add a separate RPC call for individual chunks later…? Not sure.

Reply:
I think it's fine to return whole Trie for RPC as it can't be used as an attack vector (unlike inter-node communication between validators).

Reply:
Since the RPC is trusted, there's no worry about a counter party giving us like 2GB of data only to find out at the end that it was bogus.

There are plans to introduce a trustless/MITM proof interface for querying the blockchain, but that's out of scope for this PR.